src/renderer/Dataframe.js

```diff
@@ -303,12 +330,28 @@ export default class Dataframe extends DummyDataframe {
         return features;
     }
 
-    _getFeaturesFromTriangles (geometryType, pos, viz) {
-        const point = wToR(pos.x, pos.y, {
-            center: this.center,
-            scale: this.scale
-        });
+    _computePointRadius (feature, viz) {
+        const SATURATION_PX = 1024;
+        const diameter = Math.min(viz.width.eval(feature), SATURATION_PX) + Math.min(viz.strokeWidth.eval(feature), SATURATION_PX);
+
+        return diameter / 2;
+    }
+
+    _computeLineWidthScale (feature, viz) {
+        const SATURATION_PX = 1024;
```

Comment:
It seems this constant is reused among the different methods. We could push it up to share it

Reply:
:+1: